src/components/tables/QueueTable.tsx

```diff
@@ -365,12 +380,29 @@ const QueueTable = () => {
       {({ handleArchive, hasAdminSelected, isArchiving }) => (
         <>
           <BaseTable
+            actionItems={
+              <>
+                <Button
+                  size="sm"
+                  gap={2}
+                  colorScheme="orange"
+                  variant="outline"
+                  onClick={openSuggestModal}
+                >
+                  Suggest source
```

Comment:
Then it might be better to include it in the copy of the text above

Reply:
> Then it might be better to include it in the copy of the text above

Please give me a better explanation of what you are saying here, it's not clear to me what "the text above" means.